### ISSUE TITLE
Error: HOOKSTATE-107 in React components tree

### ISSUE THREAD

**Issue by tynor** [avkonst/hookstate#90]
Hello!

It appears that the Components view in React dev tools cannot inspect components that take Hookstate State objects as properties. This trace was taken from one of the examples in Hookstate's docs:

```
Uncaught Error: Error: HOOKSTATE-107 [path: /0/name]. See https://hookstate.js.org/docs/exceptions#hookstate-107
    e https://hookstate.js.org/common.f75e6722.js:1
    e https://hookstate.js.org/common.f75e6722.js:1
    isElement moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:10702
    getDataType moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:608
    dehydrate moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:1349
    dehydrate moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:1492
    dehydrate moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:1492
    cleanForBridge moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:1085
    inspectElement moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:5376
    Agent moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:8559
    emit moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:2349
    _wallUnlisten moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:12126
    listener moz-extension://b933bfea-d804-7849-98e0-e79b322226f9/build/react_devtools_backend.js:9759
```

The `react_devtools_backend.js:10702` line is:

```js
return "object" === _typeof(a) && null !== a && a.$$typeof === b;
```

I'm not sure what the best solution here would be, since I'm not incredibly familiar with the internals of either Hookstate or React dev tools. However, it would be very nice to be able to inspect components taking State objects.

**Comment by avkonst** [comment]
I will have a look. This exception is to trap users errors. Dev tools are generic so they can not know what is right what is wrong... I will think how to address this issue.

**Comment by avkonst** [comment]
Fixed in 3.0.4
